tests/unit/inspect/test_browsers.py

```diff
@@ -0,0 +1,266 @@
+    assert isinstance(local_dimension_column.dimension, LocalDimension)
+
+    external_dimension_column = columns["ExternalDimension"]
+    assert external_dimension_column.csv_column_title == "ExternalDimension"
+    assert (
+        external_dimension_column.cell_uri_template
+        == "http://www.example.org/code-lists/regions/{+externaldimension}"
+    )
+    assert (
+        external_dimension_column.dimension.dimension_uri
+        == "http://purl.org/linked-data/sdmx/2009/dimension#refArea"
+    )
+    assert isinstance(external_dimension_column, DimensionColumn)
+    assert isinstance(external_dimension_column.dimension, ExternalDimension)
+
+    local_attribute_column = columns["LocalAttribute"]
+    assert local_attribute_column.csv_column_title == "LocalAttribute"
+    assert local_attribute_column.cell_uri_template is None
+    assert (
+        local_attribute_column.attribute.attribute_uri
+        == "testing-csvwbrowser-pivoted-shape.csv#attribute/localattribute"
+    )
+    assert local_attribute_column.attribute.label == "LocalAttribute"
+    assert isinstance(local_attribute_column, AttributeColumn)
+    assert isinstance(local_attribute_column.attribute, LocalAttribute)
+
+    external_attribute_column = columns["ExternalAttribute"]
+    assert external_attribute_column.csv_column_title == "ExternalAttribute"
+    assert (
+        external_attribute_column.cell_uri_template
+        == "http://www.example.org/status/{+externalattribute}"
+    )
+    assert (
+        external_attribute_column.attribute.attribute_uri
+        == "testing-csvwbrowser-pivoted-shape.csv#attribute/externalattribute"
+    )
+    assert isinstance(external_attribute_column, AttributeColumn)
+    assert isinstance(external_attribute_column.attribute, ExternalAttribute)
+
+    pivoted_obs_column = columns["Observations"]
+    assert pivoted_obs_column.csv_column_title == "Observations"
+    assert pivoted_obs_column.cell_uri_template is None
+    assert (
+        pivoted_obs_column.unit.unit_uri
+        == "testing-csvwbrowser-pivoted-shape.csv#unit/some-unit"
+    )
+    assert pivoted_obs_column.unit.label == "Some Unit"
+    assert (
+        pivoted_obs_column.measure.measure_uri
+        == "testing-csvwbrowser-pivoted-shape.csv#measure/some-measure"
+    )
+    assert pivoted_obs_column.measure.label == "Some Measure"
+    assert isinstance(pivoted_obs_column, PivotedObservationsColumn)
+    assert isinstance(pivoted_obs_column.unit, LocalUnit)
+    assert isinstance(pivoted_obs_column.measure, LocalMeasure)
+
+
+def test_standard_shape_browser_locally_defined_measures_units():
+    """
+    Tests whether a CsvWBrowser correctly represents the expected CSVW structure and
+    correctly extracts underlying values from the inspector functions in a standard shape
+    CSVW input with externally defined measures and units.
+    NOTE this test currently does not test anything that isn't covered by the previous test.
```

Comment:
Is this true? It tests the Standard shape observations column, measures and units columns.

Reply:
My mistake, this comment should be in the following test